app/controllers/facilities_controller.rb

```diff
@@ -138,6 +138,7 @@ def transactions
     @search = TransactionSearch::Searcher.billing_search(order_details, @search_form, include_facilities: current_facility.cross_facility?)
     @date_range_field = @search_form.date_params[:field]
     @order_details = @search.order_details.joins(order: :user).reorder(sort_clause)
+    @grand_total = order_details.sum(:actual_cost)
```

Comment:
@LeticiaErrandonea I still need to add some specs to cover the math.  Locally it seems that subtotals include some estimated costs